code/dqc/timediff.q

```diff
@@ -0,0 +1,13 @@
+\d .dqc
+
+/ Takes a table name tn and two column names ca and cb, as well as a percentage
+/ pt and a tolerance tl in milliseconds.
+timediff:{[tn;ca;cb;pt;tl]
+  .lg.o[`timediff;"Checking the time differences in columns ",(", "sv string(ca;cb))," of table ",(string tn)];
+  ot:$[tl>re:1-(sum a)%count a:?[tn;();();(>;(0D+tl*00:00:00.001);(-;ca;cb))];
```

Comment:
?[tn;();();(>;(0D+tl*00:00:00.001);(-;ca;cb))]
What is this... an exec? 
Does this work on an ondisk table? Does it need to?